### DISCUSSION TITLE
CAPI query aborted: Cmdr name unknown

### DISCUSSION THREAD

**Discussion by Hexvon** [EDCD/EDMarketConnector#1503] [Q&A] [answered]
I did not find a suitable solution to this problem in "known Issues", also untied EDCM from Frontier, but he did not request a reverse binding.
UPD: The problem is due to the launch from the administrator, I'm sorry.

**Comment by Athanasius** [comment, marked as answer]
Hopefully this was due to the extra issue I just fixed with [5.3.3](https://github.com/EDCD/EDMarketConnector/releases/tag/Release%2F5.3.3), so give that a go please.

**Reply by Hexvon** [reply]
Hmm, yes, but any information appears only after entering the character in the game itself, and if an error occurs, need restart both the game and the application. It doesn't seem to have happened before.
UPD: Also, any change in settings breaks the EDMC data update.

**Reply by Hexvon** [reply]
When updating to 5.3.3, authorization from Frontier flew off and when trying to authorize, the browser says successful authorization, but in edmc. error: unable to get the token.
![image](https://user-images.githubusercontent.com/50929311/158441004-7581a86c-ebfa-4c58-9019-be706d0a0cd1.png)
![image](https://user-images.githubusercontent.com/50929311/158441210-67ddcfa3-a5bb-4474-aba6-3270427def72.png)

**Reply by Athanasius** [reply]
Please open a [Bug Report](https://github.com/EDCD/EDMarketConnector/issues/new?assignees=&labels=bug%2C+unconfirmed&template=bug_report.md&title=) and **supply all of the information the template asks for**.  Else I'll just be guessing as to what is causing this for you.
